### ISSUE TITLE
Improve caching behavior for tiles

### ISSUE THREAD

**Issue by mojodna** [AmericanRedCross/posm#134]
Tile caching occurs within the 10MB (default) tessera cache, so it's not as useful as one would hope. Since posm-carto (and openstreetmap-carto) aren't super quick to render, an external (easily purgeable) cache would improve the user experience.

**Comment by mojodna** [comment]
The snapshot TMSes would benefit the most from this.

**Comment by hallahan** [comment]
One thing GaiaGPS does that I like is that they actually use an MBTiles as a cache. The map hits up the MBTiles, then if it doesn't get something it tries HTTP.

**Comment by mojodna** [comment]
Yup, that's basically what I'm thinking. The usual tessera config involves doing that using a CDN and/or S3, so I need to write a bit of code to fold it into tessera proper.
